docs/local_install_skaffold.md

```diff
@@ -108,81 +341,329 @@ Running the following command will do it for you (you'll be asked for your passw
 
-```
-/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-```
+**In 3 different terminal windows, in this order**:
 
-## Running the platform
+#### 1. hlf-k8s repository
 
-Using the command line, navigate to the `hlf-k8s` folder and run:
+In the `hlf-k8s` repository, please run the command `skaffold dev` (or `skaffold run` for detached mode). The platform will be ready once the terminal displays:
 
 ```sh
-skaffold dev
-````
+[network-org-2-peer-1-hlf-k8s-chaincode-install-0-4bdd4 fabric-tools] 2019-11-14 09:14:52.070 UTC [chaincodeCmd] install -> INFO 003 Installed remotely response:<status:200 payload:"OK" >
+# or
+[network-org-2-peer-1-hlf-k8s-channel-join-0-kljgq fabric-tools] 2020-02-10 10:18:02.211 UTC [channelCmd] InitCmdFactory -> INFO 001 Endorser and orderer connections initialized
+# or
+[network-org-2-peer-1-hlf-k8s-channel-join-0-kljgq fabric-tools] 2020-02-10 10:18:02.350 UTC [channelCmd] executeJoin -> INFO 002 Successfully submitted proposal to join channel
+```
 
-Then in a separate terminal, navigate to the `substra-backend` folder and run:
+![status:200 payload:"OK"](/substra/docs/img/start_hlf-k8s.png "status:200 payload:'OK'")
 
-```sh
-skaffold dev
-````
+#### 2. substra-backend repository
 
-Lastly, in a third terminal, navigate to the `substra-frontend` folder and once again run
+In the `substra-backend` repository, please run the command `skaffold dev`. The platform will be ready once the terminal displays:
 
 ```sh
-skaffold dev
+[backend-org-2-substra-backend-server-74bb8486fb-nkq6m substra-backend] INFO - 2020-02-10 10:24:42,514 - django.server - "GET /liveness HTTP/1.1" 200 2
+# or
+[backend-org-1-substra-backend-server-77cf8cb9fd-cwgs6 substra-backend] INFO - 2020-02-10 10:24:51,393 - django.server - "GET /readiness HTTP/1.1" 200 2
 ```
 
-The platform will be ready once
-- the `hlf-k8s` terminal displays `INFO 003 Installed remotely response:<status:200 payload:"OK" >`:
+![django.server readiness](/substra/docs/img/start_backend.png "django.server readiness")
 
-```
-[network-org-2-peer-1-hlf-k8s-chaincode-install-0-4bdd4 fabric-tools] 2019-11-14 09:14:52.070 UTC [chaincodeCmd] install -> INFO 003 Installed remotely response:<status:200 payload:"OK" >
+#### 3. substra-frontend repository
+
+In the `susbtra-frontend` repository, please run the command `skaffold dev`. The platform will be ready once the terminal displays:
+
+```sh
+[frontend-org-2-substra-frontend-787554fc4b-pmh2g substra-frontend] CACHING:  /login
 ```
 
-- the `backend` terminal displays `"GET /readiness HTTP/1.1" 200 2` and `"GET /liveness HTTP/1.1" 200 2` for both `org-1` and `org2`:
+![CACHING Login](/substra/docs/img/start_frontend.png "CACHING Login")
 
+Alternatively, instead of using `skaffold`, you might want to start the `substra-frontend` with [Yarn](https://yarnpkg.com/getting-started/install):
```

Comment:
I think also this section about yarn is not needed in the local install with skaffold documentation.

Starting the frontend with yarn is not related to skaffold and it targets mostly frontend developpers in my opinion.
Maybe it could be removed or moved to the substra-frontend README?

Reply:
Okay thank you for this review @samlesu , for now, I have moved this yarn part to the `Troubleshooting` section!